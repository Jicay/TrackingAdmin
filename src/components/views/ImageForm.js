import React, {Component} from "react";
import {Container} from 'reactstrap';
import Image from "./Image";
import EmailForm from "./EmailForm";

export default class ImageForm extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <Container className="image container">
                <div className='row'>
                    <Image image='916/43406628822_cffd9b699e_b'/>
                </div>
                <div className='row'>
                    <EmailForm />
                </div>
            </Container>
        );
    }
}