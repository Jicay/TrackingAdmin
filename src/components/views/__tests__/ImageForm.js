import React from 'react';
import renderer from 'react-test-renderer';
import ImageForm from "../ImageForm";

it('renders correctly', () => {
    const tree = renderer
        .create(<ImageForm/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});