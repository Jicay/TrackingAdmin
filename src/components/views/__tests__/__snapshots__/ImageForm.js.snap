// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="image container container"
>
  <div
    className="row"
  >
    <div
      className="col align-content-center"
    >
      <img
        className="img-fluid"
        src="https://live.staticflickr.com/916/43406628822_cffd9b699e_b.jpg"
      />
    </div>
  </div>
  <div
    className="row"
  >
    <form>
      <div
        className="form-group"
      >
        <label
          htmlFor="inputMail"
        >
          Email :
        </label>
        <input
          id="inputMail"
          placeholder="Saisissez votre adresse e-mail"
          type="email"
        />
      </div>
      <button
        className="btn btn-primary"
        type="submit"
      >
        Envoyer
      </button>
    </form>
  </div>
</div>
`;
